database/dbService.js

```diff
@@ -10,24 +10,34 @@ module.exports = {
             var sql = queries.insertUser;
             con.query(sql, [user.id, user.gender, user.name.givenName, user.name.familyName, user.emails[0].value, user._json.placesLived[0].value, user.phoneNumber]
             , function (err, result) {
-                con.release();                
                 if (err) throw err;
                 callback(result.insertId);
+                con.release();                                
+            });
+        });
+    },
+
+    insertActivityLog: function(ID, firstName, lastName, action) {
+        this.tryConnect().getConnection(function (err, con) {
+            var sql = queries.insertActivityLog;
+            con.query(sql, [ID, firstName, lastName, action], function (err, result) {
+                if (err) throw err;
+                con.release();
             });
         });
     },
```

Comment:
This is the function that'll insert the activity into the database.

We take in the user's ID, name and an action. Example of an action could be "User logged in successfully"